backend/service.go

```diff
@@ -71,6 +72,10 @@ func (instance *StoreService) rpc_onuninstall(client protocol.ClientInterface, a
 	}
 	return rpc.CreateSuccessResponse("started")
 }
+func (instance *StoreService) rpc_oncancel(client protocol.ClientInterface, action data.Action) string {
```

Comment:
make function name for rpc_oncancel more descriptive since cancel can have many meanings.